src/SFA.DAS.Apprenticeships.Web/Views/ChangeOfPrice/ProviderInitiatedCheckDetails.cshtml

```diff
@@ -68,6 +68,7 @@
 		<h2 class="govuk-heading-m">Now send to the employer for approval</h2>
 		<p class="govuk-body">Your payment schedule will not change until the employer approves your request.</p>
 		<form method="post" action="/provider/@Model.ProviderReferenceNumber/ChangeOfPrice/@Model.ApprenticeshipHashedId/submit">
+			@Html.AntiForgeryToken()
```

Comment:
Did we decide that putting these on every form was not needed as it's handled by the filter?